docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/Ranking.ttinclude

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+<# if (TrainerOptions != null) { #>
+<#=OptionsInclude#>
+<# } #>
+
+namespace Samples.Dynamic.Trainers.Ranking
+{
+    public static class <#=ClassName#>
+    {<#=Comments#>
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of training data points.
+            var dataPoints = GenerateRandomDataPoints(1000);
+
+            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            var trainingData = mlContext.Data.LoadFromEnumerable(dataPoints);
+
+<# if (TrainerOptions == null) { #>
+            // Define the trainer.
+            var pipeline = mlContext.Ranking.Trainers.<#=Trainer#>();
+<# } else { #>
+            // Define trainer options.
+            var options = new <#=TrainerOptions#>;
+
+            // Define the trainer.
+            var pipeline = mlContext.Ranking.Trainers.<#=Trainer#>(options);
+<# } #>
+
+            // Train the model.
+            var model = pipeline.Fit(trainingData);
+
+            // Create testing data. Use different random seed to make it different from training data.
+            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed:123));
+
+            // Run the model on test data set.
+            var transformedTestData = model.Transform(testData);
+
+            // Take the top 5 rows.
+            var topTransformedTestData = mlContext.Data.TakeRows(transformedTestData, 5);
```

Comment:
what @wschin did in his PR, he just generate 5 rows of data. What's the point of having 500 test samples if you use only 5 of them?

Reply:
Yes you are right, I don't like this approach anyways, I have filed an issue to start using `TrainTestSplit` instead of generating training and test data separately.

I will fix that for all samples at once.